ru-RU/zen-welcome.ftl

```diff
@@ -0,0 +1,26 @@
+# Эта форма исходного кода подпадает под условия Mozilla Public
+# Лицензии Mozilla Public License, v. 2.0. Если копия MPL не была распространена вместе с этим
+# файлом, вы можете получить ее по адресу http://mozilla.org/MPL/2.0/.
+
+welcome-dialog-welcome = 👋 Добро пожаловать в { -brand-short-name}
+welcome-dialog-welcome-subtext = Быстро. Красиво. Приватно.
+welcome-dialog-get-started = Приступайте
+
+welcome-dialog-import = 😍 Приводите своих любимцев на { -brand-short-name}
+welcome-dialog-import-subtext = Продолжайте с того места, где вы остановились в предыдущем браузере
+welcome-dialog-import-action = Импортируйте свои данные
+
+welcome-dialog-theme = Чувствуйте себя как дома
+welcome-dialog-theme-subtext = Персонализируйте свой опыт с помощью любимого образа!
+
+welcome-dialog-theme-header-1 = Выберите свой идеальный цвет
+welcome-dialog-theme-header-2 = Выберите свою тему
+
+welcome-dialog-search = 🔍 Выберите поисковую систему
+welcome-dialog-search-subtext = Выберите поисковую систему, которая вам больше нравится, чтобы настроить работу с сайтом!
```

Comment:
And this one is also not critical, but "tailor the browsing experience" doesn't really mean working with a single website?